staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go

```diff
@@ -47,7 +47,7 @@ var (
 			Help:           "Counter of authenticated requests broken out by username.",
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"username"},
+		[]string{"username", "authenticator"},
```

Comment:
This metric is call `authenticated_user_requests` but it is a counter and counters should be suffixed with `_total`. Would you mind renaming this?

Reply:
authenticated_user_requests_total? 

It will be a breaking change that someone's monitoring may stop working without proper changes. Is there any special requirement for such breaking changes or is it OK to leave as is?

Reply:
These are `ALPHA` metrics so it should be okay. But how old are these metrics?

Reply:
It's very old, 6 years and 8 months. It was committed on May 19, 2016 by  
https://github.com/kubernetes/kubernetes/commit/0124e234500c799b0f0a21bae18d3c921a2752b7

Reply:
Considering how old the metric is, it would be better to deprecate it and create a new one `authenticated_user_requests_total` with the new label.